plugins/item_previews/web_client/views/HierarchyWidget.js

```diff
@@ -7,20 +7,24 @@ import ItemPreviewWidget from './ItemPreviewWidget';
 
 wrap(HierarchyWidget, 'render', function (render) {
     render.call(this);
-
-    // Only on folder views:
     if (this.parentModel.resourceName === 'folder' && this._showItems) {
-        // Add the item-previews-container.
-        var element = $('<div class="g-item-previews-container">');
-        this.$el.append(element);
+        var $element = $('<div class="g-item-previews-container">');
+        this.$el.append($element);
+        this.itemPreviewView.setElement($element);
+        this.itemPreviewView.setCollection(this.itemListView.collection);
+        this.itemPreviewView.render();
+    }
+
+    return this;
+});
 
-        // Add the item preview widget into the container.
+wrap(HierarchyWidget, '_initFolderViewSubwidgets', function (base) {
```

Comment:
Logically, I think the `_initFolderViewSubwidgets` is the right place to initialize folder view specific stuff, but it works to move the initialization inside the render().